tests/ui-api-layer-acceptance-tests/internal/dex/dex.go

```diff
@@ -5,11 +5,27 @@ import (
 	"log"
 	"os"
 	"strings"
+	"testing"
 
 	"github.com/pkg/errors"
 )
 
-func IsSCIEnabled() bool {
+const sciEnabledMessage = "SCI is enabled"
+
+func SkipTestIfShould(t *testing.T) {
+	if !isSCIEnabled() {
+		return
+	}
+
+	t.Skip(sciEnabledMessage)
+}
+
+func SkipMainIfShould() {
+	log.Println(sciEnabledMessage)
+	os.Exit(0)
+}
+
+func isSCIEnabled() bool {
```

Comment:
Hm... maybe we should change it to something similar as in console tests: https://github.com/kyma-project/console/blob/master/tests/ui-tests/utils/dex.js

Reply:
good idea, but I'd rather remove this piece of code in order to always run our tests 😛